Assets/Scripts/PlayerController.cs

```diff
@@ -0,0 +1,83 @@
+﻿using UnityEngine;
+using System.Collections;
+
+public class PlayerController : MonoBehaviour {
+
+    public float hp = 100f;
```

Comment:
This seems like an excessive amount of (mostly) unrelated variables for one class to manage